index.html

```diff
@@ -45,5 +45,194 @@ <h1 class="title">I'm Mariane <br>Glad to see you!</h1>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+
+        <div class="works works2">
+            <img src="images/Snapshoot Portfolio (3).svg" alt="" class="tonic">
+            <img src="images/Snapshoot Portfolio4.svg" alt="" class="tonic-1">
+            <div>
+                <h2 class="stories">Multi-Post Stories</h2>
+
+                <ul class="description">
+                    <li class="canopy">CANOPY</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">Back End Dev</li>
+                    <li><img src="images/Counter.svg" alt="" id="counter"></li>
+                    <li class="back">2015</li>
+                </ul>
+
+                <p class="more-info">A daily selection of privately 
+                    personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <ul class="languages">
+                    <li><button class="lang" id="lan-1">html</button></li>
+                    <li><button class="lang" id="lan-2">css</button></li>
+                    <li><button class="lang" id="lan-3">javaScript</li></button></li>
+                </ul>
+
+                <button class="see" id="see-1">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-myself" id="about-myself">
+        <div class="myself">
+            <div>
+                <h2 class="about">About Myself</h2>
+                <p class="experience">Hello I'm a software developer! 
+                        I can help you build a product , feature or website Look 
+                        through some of my work and experience! If you like what you 
+                        see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+
+                <h3 class="connect-1">LET'S CONNECT</h3>
+
+                <ul class="icons-1">
+                    <li><img src="images/Vector (4).svg" class="icons-2"></li>
+                    <li><img src="images/Group.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (5).svg" class="icons-2"></li>
+                    <li><img src="images/Vector.svg" class="icons-2"></li>
+                    <li><img src="images/Vector (3).svg" class="icons-2"></li>
+                </ul>
+
+                <button class="resume" id="resume-1">Get my resume</button>
+
+            </div>
+            
+            <div class="language-container">
+
+                    <ul class="languages-below">
+                        <li class="languages-1" id="languages-23">Languages</li>
```

Comment:
- [ ] Good job adding the arrows as specified y the previous reviewer. I think you should ensure that the arrows stick to the end of the section width regardless of the screen width so as to match the figma design. 

In my opinion, I will suggest you add the arrows as image in the HTML and use display flex to align the list of technology title ad the image 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235308897-5ce14a3b-1515-44cb-8cc2-c3304933c6ac.png)

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235308952-3f3d6e5f-0af8-44e0-b679-410121bbaadb.png)

Reply:
Done